src/platform/site-wide/user-nav/sass/user-nav.scss

```diff
@@ -52,6 +52,10 @@ span.sidelines {
     align-items: center;
   }
 
+  .contact-us-link {
+    min-width: fit-content;
```

Comment:
TIL `fit-content`! But I’m not sure IE supports it? https://caniuse.com/intrinsic-width